src/test/scala/chiselTests/UIntOps.scala

```diff
@@ -172,11 +172,11 @@ class MatchedRotateLeftAndRight(w: Int = 13) extends BasicTester {
 }
 
 class UIntLitExtractTester extends BasicTester {
-  assert("b101010".U(2) === false.B)
-  assert("b101010".U(3) === true.B)
-  assert("b101010".U(100) === false.B)
-  assert("b101010".U(3, 0) === "b1010".U)
-  assert("b101010".U(9, 0) === "b0000101010".U)
+  assert("b101010".U.apply(2) === false.B)
+  assert("b101010".U.apply(3) === true.B)
+  assert("b101010".U.apply(100) === false.B)
+  assert("b101010".U.apply(3, 0) === "b1010".U)
```

Comment:
same comment as above -- inconsistent with the implementation to add the `.apply` here